southworkscom/AI
# [Enterprise Bot Template][TS] Sync Cognitive Models between C# and TypeScript

## Description
We synchronized Cognitive Models (LUIS & QnA Maker) between C# and TypeScript to ensure that they share the same behavior.
Also, we renamed files to maintain JS-style.

![image](https://user-images.githubusercontent.com/11904023/50903839-8d501480-13fd-11e9-8766-e3403abcae37.png)

![image](https://user-images.githubusercontent.com/11904023/50903843-90e39b80-13fd-11e9-96e4-038b3d820b4e.png)

## Related Issue
[Enterprise Bot Template] Sync the structure of Cognitive Models between C# and TypeScript [#523](https://github.com/Microsoft/AI/issues/523).

## Testing Steps

1. Open `AI/templates/Enterprise-Template/src/typescript/cognitiveModels` (Typescript models) and `AI/templates/Enterprise-Template/src/csharp/EnterpriseBotSample/CognitiveModels` (C# models).
2. Check the structure between them.

## Checklist
